src/routes/index.svelte

```diff
@@ -1,4 +1,8 @@
 <style>
+    :root {
+        --card-bg-color: rgba(250, 250, 250, 0.2);
+        --link-highlight: tomato;
+    }
```

Comment:
May I know what are the two? Sounds like variable, but why do we keep it as variable rather than just hardcode the value?

Reply:
CSS variables!! just in case we need to rebrand the colors, we can just change it there, and all place that use that variable will have changed.